lib/src/__testutils__/rtlUtils.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { ComponentType } from 'react';
+import { render, RenderResult } from '@testing-library/react';
+
+import '@testing-library/jest-dom';
+
+export function rtlRender<P>(Component: ComponentType<P>, minimalProps: P) {
+  return function getShallowComponent(additionalProps?: Partial<P>): RenderResult {
```

Comment:
Name of this function should be changed

Reply:
> Name of this function should be changed

Ops! Fixed here: e8d1b39cb63f1262257b2399db9972fec7877230